src/main/java/luca/task/Event.java

```diff
@@ -0,0 +1,81 @@
+package luca.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Tasks that has as specific start time and end times.
+ */
+public class Event extends Task {
+
+    /** String representation of the event start dates/times. */
+    protected LocalDateTime start;
+
+    /** String representation of the event end dates/times. */
+    protected LocalDateTime end;
+
+    /**
+     * Constructor to create a Event task.
+     *
+     * @param description text description of the Event.
+     * @param start start dates/times.
+     * @param end end dates/times.
+     */
+    public Event(String description, LocalDateTime start, LocalDateTime end) {
+        super(description, TaskType.EVENT);
+        this.start = start;
+        this.end = end;
```

Comment:
Consider changing **"start"** and **"end"** to **"startTime"** and **"endTime"**.